guides/05_tabular_data_science_and_plots/creating_a_realtime_dashboard_from_google_sheets.md

```diff
@@ -0,0 +1,147 @@
+# Creating a Real-Time Dashboard from Google Sheets
+
+Tags: TABULAR, DASHBOARD, PLOTS 
+
+[Google Sheets](https://www.google.com/sheets/about/) are an easy way to store tabular data in the form of spreadsheets. With Gradio and pandas, it's easy to read data from public or private Google Sheets and then display the data or plot it. In this blog post, we'll build a small *real-time* dashboard, one that updates when the data in the Google Sheets updates. 
+
+Building the dashboard itself with just be 9 lines of Python code using Gradio, and our final dashboard will look like this:
+
+<gradio-app space="gradio/line-plot"></gradio-app>
+
+**Prerequisites**: This Guide uses [Gradio Blocks](../01_getting_started/01_quickstart.md#blocks-more-flexibility-and-control), so make your are familiar with the Blocks class. 
+
+The process is a little different depending on if you are working with a publicly accessible or a private Google Sheet. We'll cover both, so let's get started!
+
+## Public Google Sheets
+
+If you want to build a dashboard from a Google Sheet that is public, then loading the data is very easy, thanks to the `pandas` library.
```

Comment:
link to pandas?

Reply:
I _think_ it might be better if the sentence above the numbered bullet points is more instructive, ie "To build a dashboard from a public google sheet:"